workloads/storage/pvcscale_README.md

```diff
@@ -0,0 +1,116 @@
+### Purpose
+PVC scale test is designed to create desired number of pods and attach PVC to every pod from storage backend dynamically 
+using storage storage class. 
+
+### Prerequesties
+- PVC test requieres functional storage backend and storage class. 
+- Pbench [web server](https://distributed-system-analysis.github.io/pbench/doc/server/pbench-web-server.html)
+
+### Environment variables for PVC Scale workload playbook
+
+```
+###############################################################################
+# Ansible SSH variables.
+###############################################################################
+export PUBLIC_KEY
+export PRIVATE_KEY
+
+ORCHESTRATION_USER="root"
+###############################################################################
+# RHCOS workload variables.
+###############################################################################
+export WORKLOAD_IMAGE="quay.io/openshift-scale/scale-ci-workload"
+export KUBECONFIG_FILE="path to kube config file" 
+export PBENCH_SSH_PRIVATE_KEY_FILE
+export PBENCH_SSH_PUBLIC_KEY_FILE
+export ENABLE_PBENCH_AGENTS=true
```

Comment:
Is `=true` intentional?

Reply:
can be, but better to stay in line with other parameters ( no value )